x/ecocredit/params.go

```diff
@@ -11,6 +11,8 @@ var (
 	// TODO: Decide a sensible default value
 	DefaultCreditClassFeeTokens = sdk.NewInt(10000)
 	KeyCreditClassFee           = []byte("CreditClassFee")
+	KeyAllowedClassDesigners    = []byte("AllowedClassDesigners")
+	KeyAllowlistEnabled         = []byte("AllowlistEnabled")
```

Comment:
shouldn't this be `AllowList...` (capital L)?

Reply:
No allowlist is a word in itself, synonym of whitelist see https://en.wikipedia.org/wiki/Whitelisting